File: 800/Brain's Photos/bp.c
#include <stdio.h>

// https://codeforces.com/problemset/problem/707/A

int main() {
    int r, c;
    printf("Enter the rows: ");
    scanf("%d", &r);
    printf("Enter the cols: ");
    scanf("%d", &c);
    char arr[r][c];
    int cc = 0, cb = 0;
    for(int i = 0; i < r; i++) {
        printf("Enter the data in row: %d", i);
        for(int j = 0; j < c; j++) {
            printf("Enter the data in col: %d", j);
            scanf("%d", &arr[i][j]);
        }
    }
    printf("\n");
    printf("The array is:\n");
    for(int i = 0; i < r; i++) {
        for(int j = 0; j < c; j++) {
            printf("%c", arr[i][j]); 
        }
        printf("\n"); //endl;
    }
    for(int i = 0; i < r; i++) {
        for(int j = 0; j < c; j++) {
            if(arr[i][j] == 'C' || arr[i][j] == 'K' || arr[i][j] == 'Y') {
                cc++;
            }
            else if(arr[i][j] == 'B' || arr[i][j] == 'G' || arr[i][j] == 'W') {
                cb++;
            }
        }
    }
    if(cb > cc && cc == 0) {
        printf("#BlackandWhite.\n");
    }
    if(cb < cc && cb == 0) {
        printf("#Color.\n");
    }
    if(cc < cb && cc != 0) {
        printf("#Color.\n");
    }

    return 0;
}

File: 800/Brain's Photos/bp.py
# https://codeforces.com/problemset/problem/707/A

r = int(input("Enter the rows: "))
c = int(input("Enter the columns: "))
print("Rows are: ", r)
print("Columns are: ", c)
cc = 0
cb = 0
arr = []
for i in range(r):
    col = []
    for j in range(c):
      l = input()
      col.append(l)
    arr.append(col)
print("The array is: ", arr)

# now go for checking them
for i in range(r):
  for j in range(c):
    if arr[i][j] == 'C' or arr[i][j] == 'Y' or arr[i][j] == 'K':
      cc = cc + 1
    elif arr[i][j] == 'B' or arr[i][j] == 'G' or arr[i][j] == 'W':
      cb = cb + 1
    
print("Count of colored pixels: ", cc)
print("Count of black and white and gray pixels: ", cb)



# conditions to check
if cc > cb and cb == 0:
  print("#Color.\n")
elif cc < cb and cc == 0:
  print("#BlackandWhite.\n")
elif cb > cc and cc != 0:
  print("#Color.\n")


